hw4/seminar_task_6.py

```diff
@@ -0,0 +1,8 @@
+# Задача №6 семирара. Сумма среза списка.
+
+def sum_list(lst, min_index, max_index):
+    sum_lst = sum(lst[min_index: max_index])
+    return sum_lst
+
+
+print(sum_list([1, 2, 3, 4, 5, 6, 7, 8], 3, 7))
```

Comment:
В задаче не указана очередность индексов. Если передать так: то сумма вернет 0
```suggestion
print(sum_list([1, 2, 3, 4, 5, 6, 7, 8], 7, 3))
```